src/packages/string_basic.rs

```diff
@@ -141,14 +141,12 @@ mod print_debug_functions {
 
     /// Return the empty string.
     #[rhai_fn(name = "print", name = "to_string")]
-    pub fn print_unit(ctx: NativeCallContext, unit: ()) -> ImmutableString {
```

Comment:
Better not rename parameter names because they'd go into documentation, so the documentation will have names such as `_unit` instead of `unit`.